### ISSUE TITLE
Window gets stuck on "loading" screen

### ISSUE THREAD

**Issue by algtan** [Fredolx/open-tv#19]
When I open the application, it gets stuck on a "loading" screen (see image below). I restarted my computer and tried opening the application again, but I still get the same thing.

![image](https://user-images.githubusercontent.com/52480474/233143418-828e2a1c-936e-4915-bdf0-4f94cad50913.png)

**Comment by Fredolx** [comment]
Hi!

Could you please fill in me a few more details:

- OS (incl specific distro if linux)
- Format (M3U, M3U Link, Xtream)
- Did the issue start after upgrading from a previous version?
- Any other info that could be useful

Thank you!

**Comment by algtan** [comment]
Yeah, sure!

- OS: Fedora Linux 37 (Workstation Edition)
- Format: M3U Link
- Context:
  - This was my first time installing the application (v0.9.3). It opened without any issues the first time, but when I entered the M3U link, it did not seem to work as it would return to the original screen. (I've since tested the M3U on other devices, and I think the M3U link did not work on `open-tv` due to the VPN server that I selected). Now when I open `open-tv`, it gets stuck on a loading screen.
